### ISSUE TITLE
Agraphia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10208]
**Preferred term label:**
Agraphia

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0001825

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Gerstmann syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Usage in the literature seems to vary, but in some sources Agraphia seems to refer to a loss of hte ability to write after a stroke, while dysgraphia is more used for developmental conditions, and so possibly Gerstmann syndrome should be annotated to the existing term [Dysgraphia HP:0010526](https://hpo.jax.org/app/browse/term/HP:0010526)? Do you agree with making this distinction?

**Comment by MickeySegal** [comment]
I agree that agraphia and dysgraphia are on a continuum, so I combined our terms and will use the dysgraphia name.  For Gerstmann syndrome, each terms seems to be used with about equal frequency.

**Comment by pnrobinson** [comment]
adding Agraphia as a related synonym for now
